pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -169,11 +169,7 @@ func (f *Forwarder) filterFunc(namespace string) func(interface{}) bool {
 		holder := *l.Spec.HolderIdentity
 		_, err := f.extractIP(holder)
 		if err != nil {
-			f.logger.Warn("Found invalid Lease holder identify ", *l.Spec.HolderIdentity)
-			return false
-		}
-		if p := f.getProcessor(l.Name); p != nil && p.holder == holder {
```

Comment:
Shall we add a method `getHolder` to interface bucketProcessor so that we can keep this skipping logic?

Reply:
I'll add it back for now, but this entire process should be moved to a level-based reconciler.  I have some more changes brewing.